README.md

```diff
@@ -77,7 +77,29 @@ All platforms use [TAO](https://github.com/tauri-apps/tao) to build the window,
 
 Tao uses [gtk-rs](https://gtk-rs.org/) and its related libraries for window creation and wry also needs [WebKitGTK](https://webkitgtk.org/) for WebView. So please make sure the following packages are installed:
 
-#### Arch Linux / Manjaro:
+#### Nix via NixOS
+Latest Packages for Development:
+```nix
+let
+  # Unstable Channel | Rolling Release
+  pkgs = import (fetchTarball("channel:nixpkgs-unstable")) { };
+
+  packages = with pkgs; [
+    pkg-config
+    rustc
+    cargo
+    rustfmt
+    rust-analyzer
+    webkitgtk_4_1
+  ];
```

Comment:
We should only mention the packages related to this repo, other packages are for user preference:
```suggestion
  packages = with pkgs; [
    pkg-config
    webkitgtk_4_1
  ];
```